.rubocop.yml

```diff
@@ -26,3 +26,11 @@ Lint/AmbiguousBlockAssociation:
 
 Style/Documentation:
   Enabled: false
+  
+RSpec/ExampleLength:
+  Exclude: 
+    - spec/system/*
+
+RSpec/MultipleExpectations:
+  Exclude: 
+    - spec/system/* 
```

Comment:
missing newline at end of file - I suggest configuring your IDE to take care of it for you ;)